rkt/cli_apps.go

```diff
@@ -163,3 +251,39 @@ func (ae *appExec) Type() string {
 }
 
 // TODO(vc): --mount, --set-env, etc.
+func (al *appMount) String() string {
+	var ms []string
+	for _, m := range ((*apps.Apps)(al)).Mounts {
+		ms = append(ms, m.Volume.String(), ":", m.Path)
+	}
+	return strings.Join(ms, " ")
+}
+
+func (al *appMount) Type() string {
+	return "appMount"
+}
+
+// appsVolume is for --volume flags in the form name,kind=host,source=/tmp,readOnly=true (defined by appc)
+type appsVolume apps.Apps
+
+func (al *appsVolume) Set(s string) error {
+	vol, err := types.VolumeFromString(s)
+	if err != nil {
+		return err
```

Comment:
Please add some context to this error, like `fmt.Errorf("invalid value %q in --volume flag: %v", s, err)`